etp-backend/src/test/clj/solita/etp/service/laskutus_test.clj

```diff
@@ -66,11 +66,14 @@
      :yritykset (zipmap yritys-ids yritys-adds)
      :energiatodistukset (zipmap energiatodistus-ids energiatodistus-adds)}))
 
-;; There are 10 energiatodistus.
+;; There are 11 energiatodistus.
```

Comment:
Testidataan lisätty yksi todistus lisää.

Todistus 11 ei ole allekirjoitettu ollenkaan.
Todistus 10 on allekirjoitettu vuosia ennen todistusta 1, jolla sitten todistus 10 on korvattu. Näin ollen todistusta 10 ei saa poimia laskutukseen, mutta todistus 1 täytyy edelleen.

Reply:
Kirjoitanpa testidatan uudestaan koko featurelle niin että sitä on helpompi ymmärtää, ja tulee testattua muutama case kattavammin.